stdlib/universe/universe.flux

```diff
@@ -295,20 +287,19 @@ tripleEMA = (n, tables=<-) =>
 		|> drop(columns: ["__ema1", "__ema2"])
 
 // truncateTimeColumn takes in a time column t and a Duration unit and truncates each value of t to the given unit via map
-// Change from _time to timeColumn once Flux Issue 1122 is resolved
+// Change from _time to timeColumn once https://github.com/influxdata/flux/issues/1122 is resolved
 truncateTimeColumn = (timeColumn="_time", unit, tables=<-) =>
     tables
         |> map(fn:(r) => ({r with _time: date.truncate(t: r._time, unit: unit)}))
 
-// kaufmansER computes Kaufman's Efficiency Ratios of the `_value` column
-kaufmansER = (n, tables=<-) =>
-    tables
-        |> chandeMomentumOscillator(n: n)
-        |> map(fn:(r) => ({r with _value: (math.abs(x: r._value)/100.0)}))
-
 toString   = (tables=<-) => tables |> map(fn:(r) => ({r with _value: string(v:r._value)}))
 toInt      = (tables=<-) => tables |> map(fn:(r) => ({r with _value: int(v:r._value)}))
 toUInt     = (tables=<-) => tables |> map(fn:(r) => ({r with _value: uint(v:r._value)}))
 toFloat    = (tables=<-) => tables |> map(fn:(r) => ({r with _value: float(v:r._value)}))
 toBool     = (tables=<-) => tables |> map(fn:(r) => ({r with _value: bool(v:r._value)}))
 toTime     = (tables=<-) => tables |> map(fn:(r) => ({r with _value: time(v:r._value)}))
+
+
+simpleHeatIndex = (tables=<-) => tables 
```

Comment:
Does something like this belong in a package? Maybe a `physics` package or something? Not sure what a good name is.

Reply:
I've actually added a section to the SPEC.md called **Physical Environment** and placed this under it. I was about to also add `idealGasLaw` to the same section, with the accompanying flux function. At some point I'll also add the *complete* heat index calculation uncluding the Rothfusz regression but that will take more time.